systemd/ua-reboot-cmds.service

```
@@ -0,0 +1,10 @@
+[Unit]
+Description=Ubuntu Advantage reboot cmds
+
```

I think we need a ConditionPathExists=/var/lib/ubuntu-advantage/marker-reboot-cmds-required here so this is only included if the marker file exists?
```suggestion
ConditionPathExists=/var/lib/ubuntu-advantage/marker-reboot-cmds-required

```